jkube-kit/enricher/generic/src/test/java/org/eclipse/jkube/enricher/generic/openshift/AutoTLSEnricherTest.java

```diff
@@ -62,6 +68,10 @@
 
     @Test
     public void testAdapt() {
+        Map annotations = new LinkedHashMap<String, String>();
```

Comment:
Where is this map used? I see you're creating it but while asserting annotations you're simply doing it with `annotationkey` and `annotationvalue`

Reply:
You are right. Not used. I have pushed a change.